lessons/Early-Math/evaluating-exponents/langs/en-us/pages/exponents-variables-table.adoc

```diff
@@ -0,0 +1,69 @@
+= Variable Expressions with Exponents
+
+++++
+<style>
+table {grid-auto-rows: 1fr;}
+</style>
+++++
+
+
+Create a Circle for the given expression. Once you have drawn a Circle, evaluate the expression by substituting in the value provided in the third column. The first one is done for you.
+
+[.FillVerticalSpace, cols="^.^1a,^.^2a,^.^5a,^.4a", stripes="none", options="header"]
+|===
+| 	 | Expression	| Circle | Evaluate
+
+
+| @n
+| @math{3x^2}
+| @show{(coe '(* 3 (num-expt x 2)))}
+| Evaluate for @math{x=5}.
+
+@math{3x^2=75}
+
+
+| @n
+| @math{\frac{m^2}{4}}
+| @showsoln{(coe '(/ (num-expt m 2) 4))}
+| Evaluate for @math{m=10}.
+
+@ifsoln{@math{\frac{m^2}{4} = 25}}
+
+
+| @n
+| @math{6 + w^3}
+| @showsoln{(coe '(+ 6 (num-expt w 3)))}
+| Evaluate for @math{w=3}.
+
+@ifsoln{@math{6 + w^3 = 33}}
+
+
+| @n
+| @math{\frac{1}{25} \times 5^b}
+| @showsoln{(coe '(* 1/25 (num-expt 5 b)))}
+| Evaluate for @math{b=3}.
+
+@ifsoln{@math{\frac{1}{25} \times 5^b = 5}}
+
+
+| @n
+| @math{(7 + c)^2}
+| @showsoln{(coe '(num-expt (+ 7 c) 2))}
+| Evaluate for @math{c=13}.
+
+@ifsoln{@math{(7 + c)^2 = 400}}
+
+
+| @n
+| @math{5w^m}
+| @showsoln{(coe '(* 5 (num-expt w m)))}
+| Evaluate for @math{w=6} and @math{m=2}.
+
+@ifsoln{@math{5w^m = 180}}
+
+
+
+
+|===
+
```

Comment:
I may have missed something, but I don't think you're asking students to substitute values into the variables in these expressions without ever having introduced them to the concept. In. my classroom, this would definitely require some scaffolding. If I had taught it before I would want to go over an example as a whole class. But I could also imagine not having taught it before teaching a lesson on exponents... which makes me wonder this page should be an `@opt-printable-exercise` at the moment for teachers whose students are prepared for it.